source/features/bugs-tab.tsx

```diff
@@ -87,7 +86,7 @@ async function init(): Promise<void | false> {
 	bugsCounter.title = '';
 
 	// Update Bugs’ link
-	new SearchQuery(bugsLink).add('label:bug');
+	new SearchQuery(bugsTab as HTMLAnchorElement).add('label:bug');
```

Comment:
Can you drop this type cast once you change the `issuesTab` selector to `a.UnderlineNav-item[data-hotkey="g i"]` (i.e. including the tag name `a`)?

Reply:
I thought so too, but unfortunately it seems that `cloneNode` loses the element type: https://github.com/Microsoft/TypeScript/issues/283

Reply:
Seems like the workaround added in #2670 doesn't work in this case?

Reply:
No it does actually, I just forgot the return value of `elementReady()` needed to be coerced. My bad.

Reply:
There’s an issue open on the element-ready repo to implement the selector parser for types, if anyone is interested. Relatively easy PR for that